ulog/builder_test.go

```diff
@@ -147,6 +148,22 @@ func TestConfiguredLoggerWithSentryError(t *testing.T) {
 	testSentry(t, "invalid_dsn", false)
 }
 
+func TestMetricsHook(t *testing.T) {
+	// TODO(glib): logging init needs to be fixed
+	// Any tests that run in development environment can't configure the logger,
+	// it just creates the default one...
+	defer env.Override(t, config.EnvironmentKey(), "wat?")()
+
+	s, r := metrics.NewTestScope()
+	l := Builder().WithScope(s).Build()
+
+	r.Cw.Add(1)
+	l.Warn("Warning log!")
+	r.Cw.Wait()
+
+	assert.Equal(t, 1, len(r.Counters))
```

Comment:
That is tested elsewhere, I think it would be redundant. The test here is to make sure that we set up the hook at all